libraries/AP_NavEKF/EKFGSF_yaw.cpp

```diff
@@ -176,16 +132,55 @@ void EKFGSF_yaw::update(const Vector3f &delAng,
         float yawDelta = wrap_PI(EKF[mdl_idx].X[2] - GSF.yaw);
         GSF.yaw_variance +=  GSF.weights[mdl_idx] * (EKF[mdl_idx].P[2][2] + sq(yawDelta));
     }
-
-    // prevent the same velocity data being used again
-    vel_data_updated = false;
 }
 
-void EKFGSF_yaw::pushVelData(Vector2f vel, float velAcc)
+void EKFGSF_yaw::fuseVelData(Vector2f vel, float velAcc)
```

Comment:
it's not a big deal but you we can reduce some copying of memory by changing "Vector2f vel" to "const Vector2f &vel".  This changes it from "pass by value" to "pass by reference" but it is still "constant" so you don't need to worry about accidentally changing it within the function, that won't happen.

Reply:
Fixed